crates/sncast/tests/e2e/script/call.rs

```diff
@@ -1,15 +1,17 @@
 use crate::helpers::constants::{SCRIPTS_DIR, URL};
```

Comment:
the same comment as above - let's maybe avoid creating tempdirs in unnecessary places like call etc. We should only use tempdirs, when we need to alter fs or depend on some salted contracts/data

Reply:
ahh shit, this is `script call` test, and my dumb ass thought this was `subcommand call` test. Sorry, you're right. This should be left as is.